src/modules/auth/resources/ResourcesDeletedDetailContainer.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import {
+	Container, Row, Col, Card,
+	CardHeader, CardBody, Button,
+	CardFooter, ButtonGroup
+} from 'reactstrap';
+
+import { DateTime, ButtonWithAuthz } from 'asab-webui';
+
+const ResourceDetailContainer = (props) =>  {
```

Comment:
@petrKavulok Why it has a name `ResourceDetailContainer`, when you are in `ResourcesDeletedDetailContainer` (this is also kinda tricky naming, please find better)